lib/diagnostic.js

```diff
@@ -32,7 +36,8 @@ db.dropDatabase();
 // Jon Arryn
 
 // your code begins here
-
+db.houses.update({House: "Stark"}, {$set: {members: ['Ned Stark', 'Arya Stark',
```

Comment:
Each member is supposed to be added to their respective houses.  This adds all of them to House Stark.  `$push` is also supposed to be used at least once.

```javascript
db.houses.update({
  name: 'House Stark'
}, {
  $set: {
    members: ['Ned Stark']
  }
});

db.houses.update({
  name: 'House Targaryen'
}, {
  $set: {
    members: ['Viserys Targaryen']
  }
});

db.houses.update({
  name: 'House Arryn'
}, {
  $set: {
    members: ['Jon Arryn']
  }
});

db.houses.update({
  name: 'House Stark'
}, {
  $push: {
    members: {
      $each: ['Arysa Stark', 'Sansa Stark']
    }
  }
});

db.houses.update({
  name: 'House Targaryen'
}, {
  $push: {
    members: 'Daenerys Targaryen'
  }
});
```